test/spec/features/auto-place/AutoPlaceSpec.js

```diff
@@ -176,67 +177,169 @@ describe('features/auto-place', function() {
 
   describe('util', function() {
 
-    describe('#deconflictPosition', function() {
+    describe('#findFreePosition', function() {

```

Comment:
From `bpmn-js` perspective this is a breaking change. Did you consider implement it in a non breaking fashion? 

Alternatively, what is the proposed rollout strategy?

Reply:
It is. The result would be a major release of diagram-js. Alternatively, we could decide to not change the name.